website/src/documentation/transforms/python/element-wise/tostring.md

```diff
@@ -19,9 +19,38 @@ limitations under the License.
 -->
 
 # ToString
+
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
+
 Transforms every element in an input collection a string.
 
-## Examples
-See [BEAM-7389](https://issues.apache.org/jira/browse/BEAM-7389) for updates. 
+## Example
+
+Any non-string element can be converted to a string using sandard Python functions and methods.
```

Comment:
@aaltay Should we comment that this is not a Beam transform for python? It's obvious if you look at the example, but my concern is that people would hunt around for the pydoc

Reply:
Sure, if you thing that people will be confused. Just note that, a transform would not be necessary and standard functions could be used.

Reply:
This page might be most useful to people who are familiar with the Java SDK and are looking for the equivalent Python transforms, which would allow them to do a Google search on them too.

I also think it's fairly obvious from the code sample that there is no transform needed.

For simplicity, how about we leave it as it is?

Reply:
SG for now, we'll also need to think about how this class of colabs would change when their corresponding python transforms are added.